src/installer/records.py

```diff
@@ -213,5 +213,9 @@ def parse_record_file(rows: Iterable[str]) -> Iterator[Tuple[str, str, str]]:
             )
             raise InvalidRecordEntry(elements=elements, issues=[message])
 
+        # Convert Windows paths to use / for consistency
+        if os.sep == "\\":
+            elements[0] = elements[0].replace("\\", "/")  # pragma: no cover
```

Comment:
```suggestion
        # Convert Windows paths to use / for consistency
        elements[0] = elements[0].replace("\\", "/")
```

It'd be better to consistently replace `\` with `/`, regardless of OS. Otherwise, a wheel built on Windows won't be installable on other platforms; which is what we want to solve here.